docs: setup issue template (#453)
vercel/swr: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report

> about: Create a bug report for the Next.js core / examples

Reviewer: ```suggestion
about: Create a bug report for the SWR library
```

Reviewer 2: Great great catch!

Author: now you know this line is from next.js repo 🙈

---

# Bug report

## Description / Observed Behavior

> what kind of issues did you encounter with swr?

Reviewer 2: ```suggestion
What kind of issues did you encounter with SWR?
```

## Expected Behavior

> what did you expect swr to behave here?

Reviewer 2: ```suggestion
What did you expect SWR to behave here?
```

> what did you expect swr to behave here?

Reviewer: ```suggestion
How did you expect SWR to behave here?
```

## Repro Steps / Code Example

> Or share your code snippet or a codesandbox link is also appreciated!

Reviewer 2: ```suggestion
Or share your code snippet or a [CodeSandbox](https://codesandbox.io) link is also appreciated!
```

## Additional Context

> swr version. 

Reviewer 2: ```suggestion
SWR version. 
```
